modules/ROOT/Edits/create-endpoint.adoc

```diff
@@ -0,0 +1,99 @@
+= Create and Configure Endpoints
+
+You can create new endpoints from the *Host* or *Partner* page, or by using the Message Flow wizard.
```

Comment:
Note that this text introduces first Host and then Partner, but their order is switched in the narratives that follow.